docs/standards/VEP/vep-1155.md

```diff
@@ -0,0 +1,538 @@
+---
+title: "VEP-1155: Multi-Token"
+---
+
+```preamble
+VEP: 1155
+author: Evgeny Shatalov <evgeny.a.shatalov@gmail.com>, Aleksei Kolchanov <a.kolchanov@numi.net>
+status: Review
+type: Informational
+created: 2023-03-08
+requires: TIP-6
```

Comment:
Are you sure that only TIP-6? You are mentioning TIP-3 and TIP-4 actually

Reply:
And not only just by mentioning, actually you use 'em while describe an implementing